tests/models/univnet/test_modeling_univnet.py

```diff
@@ -0,0 +1,356 @@
+
+    @unittest.skip(reason="UnivNetGan.forward does not accept an inputs_embeds argument.")
+    def test_inputs_embeds(self):
+        pass
+
+    @unittest.skip(reason="UnivNetGan does not use input embeddings and thus has no get_input_embeddings method.")
+    def test_model_common_attributes(self):
+        pass
+
+    @unittest.skip(reason="UnivNetGan does not support all arguments tested, such as output_hidden_states.")
+    def test_model_outputs_equivalence(self):
+        pass
+
+    @unittest.skip(reason="UnivNetGan does not output hidden_states.")
+    def test_retain_grad_hidden_states_attentions(self):
+        pass
+
+    def test_batched_inputs_outputs(self):
+        config, inputs = self.model_tester.prepare_config_and_inputs_for_common()
+
+        for model_class in self.all_model_classes:
+            model = model_class(config)
+            model.to(torch_device)
+            model.eval()
+
+            batched_spectrogram = inputs["input_features"].unsqueeze(0).repeat(2, 1, 1)
+            batched_noise_sequence = inputs["noise_sequence"].unsqueeze(0).repeat(2, 1, 1)
+            with torch.no_grad():
+                batched_outputs = model(
+                    batched_spectrogram.to(torch_device),
+                    batched_noise_sequence.to(torch_device),
+                )
+
+            self.assertEqual(
+                batched_spectrogram.shape[0],
+                batched_outputs.shape[0],
+                msg="Got different batch dims for input and output",
+            )
+
+    def test_unbatched_inputs_outputs(self):
+        config, inputs = self.model_tester.prepare_config_and_inputs_for_common()
+
+        for model_class in self.all_model_classes:
+            model = model_class(config)
+            model.to(torch_device)
+            model.eval()
+
+            with torch.no_grad():
+                outputs = model(inputs["input_features"].to(torch_device), inputs["noise_sequence"].to(torch_device))
+            self.assertTrue(outputs.dim() == 1, msg="Got un-batched inputs but batched output")
+
+    def test_unbatched_batched_outputs_consistency(self):
+        config, inputs = self.model_tester.prepare_config_and_inputs_for_common()
+
+        for model_class in self.all_model_classes:
+            model = model_class(config)
+            model.to(torch_device)
+            model.eval()
+
+            unbatched_spectrogram = inputs["input_features"].detach().clone()
+            unbatched_noise_sequence = inputs["noise_sequence"].detach().clone()
+            batched_spectrogram = inputs["input_features"].unsqueeze(0)
+            batched_noise_sequence = inputs["noise_sequence"].unsqueeze(0)
+
+            with torch.no_grad():
+                unbatched_outputs = model(
+                    unbatched_spectrogram.to(torch_device),
+                    unbatched_noise_sequence.to(torch_device),
+                )
+                print(f"Unbatched outputs shape: {unbatched_outputs.shape}")
```

Comment:
(nit) we can remove these print statements -> the `msg` in the check at the end of this test will tell us if there are any inconsistencies